ILSpy/TreeNodes/EventTreeNode.cs

```diff
@@ -45,14 +46,21 @@ public EventTreeNode(IEvent @event)
 
 		public IEvent EventDefinition { get; }
 
-		public override object Text => GetText(EventDefinition, this.Language) + EventDefinition.MetadataToken.ToSuffixString();
+		public override object Text => GetText(GetEventDefinition(), this.Language) + EventDefinition.MetadataToken.ToSuffixString();
+
+		private IEvent GetEventDefinition()
+		{
+			return ((MetadataModule)EventDefinition.ParentModule.PEFile
+				?.GetTypeSystemWithCurrentOptionsOrNull()
+				?.MainModule).GetDefinition((EventDefinitionHandle)EventDefinition.MetadataToken);
```

Comment:
`(a?.b).c()` is likely to result in `NullReferenceException`s.
It would be better to use the original member definition if the `OrNull` suddenly starts returning `null`